src/pages/index.astro

```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
         />
 
         <Title level={3} className="font-bold text-3xl mb-4 mt-12 lg:mt-auto">
-          EuroPython 2023 Photos
+          On-site Volunteer
         </Title>
 
         <ButtonLink
-          href="https://www.flickr.com/photos/europython/collections/72157722021442529/"
+          href="/volunteers"
         >
-          Highlights of EuroPython 2023
+          Join us in Shaping EuroPython 2024
```

Comment:
I expect the uppercase "Shaping" to be on purpose?

Reply:
Yes, but I also wanted to make us into Us. Now fixed. Thanks for spotting it.